src/net.h

```diff
@@ -1234,6 +1234,9 @@ class CNode
     void MaybeSetAddrName(const std::string& addrNameIn);
 
     std::string ConnectionTypeAsString() const;
+
+    /** Whether this peer is an inbound onion, e.g. connected via our Tor onion service. */
```

Comment:
nit question: should this say "i.e."? I mean are there any other ways to establish an inbound onion other than via the tor onion service?

Reply:
I don't mind updating this and taking @vasild's suggestion at the same time.

Reply:
Can do as a follow-up. The merge script is already running and this should land in master in the next couple of minutes.

Reply:
Oh ok, will do in #20210.

Reply:
> nit question: should this say "i.e."? I mean are there any other ways to establish an inbound onion other than via the tor onion service?

Done in #21167